tests/grid/extrude_orientation_03.cc

```diff
@@ -0,0 +1,69 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2005 - 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+// Test GridGenerator::extrude_triangulation taking slice z-coordinate values.
+// This test is just a replicate of the one in extrude_orientation_02.cc except
+// that the newly created overload of GridGenerator::extrude_triangulation is
+// tested for face and edge orientations for the resulting 3d mesh.
+// See https://github.com/dealii/dealii/issues/6158.
+//
+// test this for a circle extruded to a cylinder
+
+#include "../tests.h"
+#include <deal.II/grid/tria.h>
+#include <deal.II/grid/grid_generator.h>
+
+void test(std::ostream &out)
```

Comment:
Are you actually using `out`?

Reply:
Oh, I didn't. Actually, I copy-pasted from extrude_orientation_02.cc for this section. I will just modify that one as well.

Reply:
done and done for extrude_orientation_02.cc as well